leoism/step/portfolio/src/main/webapp/script.js

```diff
@@ -66,11 +66,18 @@ function createComment(commentInformation) {
 function loadComments() {
   const responsePromise = fetch('/data');
   const commentsContainer = document.getElementById('comments-container');
-  const amtCurrentlyDisplayed = parseInt(commentsContainer.getAttribute('data-display'));
+  const amtCurrentlyDisplayed =
+      parseInt(commentsContainer.getAttribute('data-display'));
 
   const nextTenComments = amtCurrentlyDisplayed + 10;
   responsePromise.then((response) => response.json()).then((comments) => {
     for (let i = amtCurrentlyDisplayed; i < nextTenComments; i++) {
+      // Make load more button not appear when all comments have been loaded.
+      if (i === comments.length - 1) {
+        const loadMoreButton = document.getElementById('load-more-button');
+        loadMoreButton.style.display = 'none';
+      }
+
       if (comments[i] === undefined) {
         commentsContainer.setAttribute('data-display', i);
         break;
```

Comment:
On line 88, since this if statement is only executed on the last iteration, can this go outside of the loop?

Reply:
I created https://github.com/googleinterns/step155-2020/pull/40 to address this comment. Since, this PR is closed and merged it's not possible to add more commits (based on the research I did). If there is another way of doing this let me know